src/pages/Home/components/Days/Tabs.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import {TabsStyle, TabsWrapper,TabStyle,Cancel} from "./DaysStyles";
+
+interface Props{
+}
+
+export const Tabs =(props:Props)=>{
+  const tabs =[
+    {
+      value:'На тиждень',
+    },
+    {
+      value:'На 10 днів',
+    },
+    {
+      value:'На місяць',
+    },
+  ];
+  return (
+  <TabsStyle>
+    <TabsWrapper>
+      {tabs.map((tab) => (
+        <TabStyle key={tab.value}>
```

Comment:
It's better to call Button with props tab
```suggestion
        <Button
          tab
          key={tab.value}
          >
```